src/constants.js

```diff
@@ -172,19 +172,10 @@ export const SAUCE_CONNECT_CLI_PARAMS = [{
 }, {
     name: 'dns',
     description: 'Use specified name server. To specify multiple servers, separate them with comma. Use IP addresses, optionally with a port number, the two separated by a colon. Example: --dns 8.8.8.8,8.8.4.4:53'
-}, {
-    name: 'doctor',
-    description: 'Perform checks to detect possible misconfiguration or problems.',
-    type: 'boolean'
```

Comment:
Why are we removing these (without first deprecating them)?

Reply:
What do you mean by "There is no way anyone can use this flag with the node client"?  Does it just break, or something?

Reply:
this is a debugging option for SC client, it was meant for SC standalone binary to debug DNS, connectivity, etc...
It's no longer maintained, and its output is fairly useless. Another `doctor` subcommand is in the works.